### ISSUE TITLE
Reduce schemas payload size 

### ISSUE THREAD

**Issue by Markionium** [dhis2/d2#77]
From @larshelge 
> Currently, the request for /schemas.json returns 1.1 Mb raw uncompressed text. This leads to too long load time on slow lines where compression is not enabled. The app should use field filtering to only include the fields under "properties" which are actually in use.

**Comment by nicolayr** [comment]
Support optionally loading specific list of schemas/models.

**Comment by Markionium** [comment]
Specifying specific schemas is done as follows:

```js
d2.init({ 
  baseUrl: 'http://localhost:8080/dhis/api', 
  schemas: ['dataSet', 'user'] }
)
  .then((d2) => {
    // d2.models will only have dataSet and user as available models to work with.
  });
```

When recalling init the old schemas stay present on the models object, due to `ModelDefinitions`  being a singleton. I do not see any immediate problem with this however, so we might as well keep it this way for now.

Note, trying to use this as a incremental loading of schemas is not the right way, as it would also re-init the rest of the library, reloading currentUser, systemSettings etc.

**Comment by Markionium** [comment]
Available in 24.1 versions
